vizro-core/src/vizro/models/_navigation/nav_link.py

```diff
@@ -57,10 +58,14 @@ def build(self, *, active_page_id=None):
 
         button = dbc.Button(
             [
-                html.Span(self.icon, className="material-symbols-outlined"),
-                # TODO: commented out until we insert styling for the tooltip or find a better way to display it (e.g.
-                # try dbc.Popover or Dash mantine components tooltip?).
-                # dbc.Tooltip(html.P(self.label), target=self.id, placement="bottom", className="custom-tooltip"),
+                dmc.Tooltip(
+                    label=self.label,
+                    offset=6,
```

Comment:
Could you double-check the CSS applied to the tooltip? It seems they are not aligned with the designs that are in Figma? Or did D. give you other designs? 

In Figma/Storybook, it would be good if you double-check the offset, zIndex and the colors. On dark theme e.g. the tooltip is white, while on light theme the tooltip is dark. Currently it's the other way around.